lib/chef/resource/windows_path.rb

```diff
@@ -24,20 +24,12 @@ class Resource
     #
     # @since 13.4
     class WindowsPath < Chef::Resource
-
+      resource_name :windows_path
       provides :windows_path, os: "windows"
```

Comment:
should drop the "os: windows" bit here.

Reply:
I was planning to nuke those in Chef 14. I'm certain there's people out there coding around the fact that these are no-ops on non-windows platforms. We can nuke them all out then and everyone is happy.

Reply:
they're not no-ops, though, they blow up hard.

Reply:
and i think right now chefspec testing of windows_path on anything other than windows might be problematic -- sort of a bug we should fix in chefspec, but its easy enough to fix it so that even on linux or whatever we can resolve windows_path to a resource.

Reply:
I thought chefspec was one of the big reasons we kept all this stuff the way it was.